openreview/api/client.py

```diff
@@ -65,7 +65,6 @@ def __init__(self, baseurl = None, username = None, password = None, token= None
         self.group_edits_url = self.baseurl + '/groups/edits'
         self.user_agent = 'OpenReviewPy/v' + str(sys.version_info[0])
 
-        self.limit = 1000
```

Comment:
this property is being used inside the tools.py module to make concurrent requests:

https://github.com/openreview/openreview-py/blob/master/openreview/tools.py#L686

Reply:
I see, I missed that. Closing.